### PR TITLE
555: Implement RSS feed to Low Stock Notifications

### PR DESCRIPTION
**Pull request by RomaKis**
### Fixed Issues (if relevant)
1. https://github.com/magento-engcom/msi/issues/555.

### CODE REVIEW

File: app/code/Magento/InventoryLowQuantityNotification/Model/ResourceModel/Rss/NotifyStock/GetCollection.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+     * @param ManagerInterface $eventManager
+     * @param SelectBuilder $selectBuilder
+     */
+    public function __construct(
+        CollectionFactory $collectionFactory,
+        ManagerInterface $eventManager,
+        SelectBuilder $selectBuilder
+    ) {
+        $this->collectionFactory = $collectionFactory;
+        $this->eventManager = $eventManager;
```
Comment (naydav) [inline app/code/Magento/InventoryLowQuantityNotification/Model/ResourceModel/Rss/NotifyStock/GetCollection.php]: Unused variable

File: app/code/Magento/InventoryLowQuantityNotification/Model/ResourceModel/Rss/NotifyStock/GetCollection.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+namespace Magento\InventoryLowQuantityNotification\Model\ResourceModel\Rss\NotifyStock;
+
+use Magento\Framework\Event\ManagerInterface;
+use Magento\Inventory\Model\ResourceModel\SourceItem\Collection;
+use Magento\Inventory\Model\ResourceModel\SourceItem\CollectionFactory;
+use Magento\InventoryApi\Api\Data\SourceItemInterface;
+use Magento\InventoryLowQuantityNotification\Model\ResourceModel\Rss\NotifyStock\SelectBuilder;
+
+class GetCollection
```
Comment (naydav) [inline app/code/Magento/InventoryLowQuantityNotification/Model/ResourceModel/Rss/NotifyStock/GetCollection.php]: Pls provide more clear name and some description

File: app/code/Magento/InventoryLowQuantityNotification/etc/di.xml [hunk 3]
```diff
@@ -14,4 +14,18 @@
                 type="Magento\InventoryLowQuantityNotification\Model\SourceItemConfiguration\Command\Get"/>
     <preference for="Magento\InventoryLowQuantityNotificationApi\Api\DeleteSourceItemConfigurationInterface"
                 type="Magento\InventoryLowQuantityNotification\Model\SourceItemConfiguration\Command\Delete"/>
+    <type name="Magento\Catalog\Block\Adminhtml\Rss\NotifyStock\DescriptionProvider">
+        <plugin name="rss_adapt_description_provider"
+                type="Magento\InventoryLowQuantityNotification\Plugin\Rss\AdaptDescriptionProvider"/>
+    </type>
+    <!-- LowStockCondition -->
```
Comment (naydav) [inline app/code/Magento/InventoryLowQuantityNotification/etc/di.xml]: Looks like we don't need this configuration

File: app/code/Magento/InventoryLowQuantityNotificationApi/Test/_files/enable_manage_stock_for_products.php [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+$skus = ['SKU-1', 'SKU-2', 'SKU-3'];
+
+foreach ($skus as $sku) {
+    $product = $productRepository->get($sku);
+
+    $stockItem = $product->getExtensionAttributes()->getStockItem();
+    $stockItem->setUseConfigManageStock(false);
+    $stockItem->setManageStock(true);
+    $stockItem = $product->getExtensionAttributes()->setStockItem($stockItem);
```
Comment (naydav) [inline app/code/Magento/InventoryLowQuantityNotificationApi/Test/_files/enable_manage_stock_for_products.php]: Save legacy StockItem via `\Magento\CatalogInventory\Api\StockItemRepositoryInterface`

File: app/code/Magento/InventoryLowQuantityNotification/etc/di.xml [hunk 5]
```diff
@@ -14,4 +14,18 @@
                 type="Magento\InventoryLowQuantityNotification\Model\SourceItemConfiguration\Command\Get"/>
     <preference for="Magento\InventoryLowQuantityNotificationApi\Api\DeleteSourceItemConfigurationInterface"
                 type="Magento\InventoryLowQuantityNotification\Model\SourceItemConfiguration\Command\Delete"/>
+    <type name="Magento\Catalog\Block\Adminhtml\Rss\NotifyStock\DescriptionProvider">
```
Comment (naydav) [inline app/code/Magento/InventoryLowQuantityNotification/etc/di.xml]: Looks like we don't need this configuration

**Review by naydav — CHANGES_REQUESTED** [review]

File: app/code/Magento/InventoryLowQuantityNotification/Model/ResourceModel/Rss/NotifyStock/GetCollection.php (hunk 1 above)
Comment (RomaKis) [reply app/code/Magento/InventoryLowQuantityNotification/Model/ResourceModel/Rss/NotifyStock/GetCollection.php]: Done.

File: app/code/Magento/InventoryLowQuantityNotification/Model/ResourceModel/Rss/NotifyStock/GetCollection.php (hunk 2 above)
Comment (RomaKis) [reply app/code/Magento/InventoryLowQuantityNotification/Model/ResourceModel/Rss/NotifyStock/GetCollection.php]: Fixed.

File: app/code/Magento/InventoryLowQuantityNotification/etc/di.xml (hunk 3 above)
Comment (RomaKis) [reply app/code/Magento/InventoryLowQuantityNotification/etc/di.xml]: Yes. Fixed.

File: app/code/Magento/InventoryLowQuantityNotificationApi/Test/_files/enable_manage_stock_for_products.php (hunk 4 above)
Comment (RomaKis) [reply app/code/Magento/InventoryLowQuantityNotificationApi/Test/_files/enable_manage_stock_for_products.php]: Done.

File: app/code/Magento/InventoryLowQuantityNotification/etc/di.xml (hunk 5 above)
Comment (RomaKis) [reply app/code/Magento/InventoryLowQuantityNotification/etc/di.xml]: Done

**Pull request merged**